### PR TITLE
Update 01b_dplyr-manipulating-data.Rmd

### PR DESCRIPTION
**Pull request by jmerodriguez**

### CODE REVIEW

File: slides/01b_dplyr-manipulating-data.Rmd [hunk 1]
```diff
@@ -334,6 +334,28 @@ Error: Column `mean_price` must be length 1 (a summary value), not 561
 
 Get number of observations
 
+## piping `dplyr` verbs together
+
+`dplyr`verbs can be piped together in any order you want, although different orders can give you different results, so be careful!
+
+```{r}
+
+texas_housing_data %>%
```
Comment (anisfeld) [inline slides/01b_dplyr-manipulating-data.Rmd:343]: This is great. Can you add `na.rm = TRUE` both times you call `mean()`?

Comment (anisfeld) [reply slides/01b_dplyr-manipulating-data.Rmd:343]: @jmerodriguez

File: slides/01b_dplyr-manipulating-data.Rmd [hunk 2]
```diff
@@ -334,6 +334,29 @@ Error: Column `mean_price` must be length 1 (a summary value), not 561
...
+  select(city, year, month, sales, volume) %>%
+  mutate(log_mean_price = log(volume / sales)) %>%
+  filter(year == 2013) %>%
+  summarize(log_mean_price_2013 = mean(log_mean_price,
```
Comment (jmerodriguez) [inline slides/01b_dplyr-manipulating-data.Rmd:347]: I changed what you asked for and knit

**Review by jmerodriguez — COMMENTED** [review]
I knit the new slides, but my R is showing the packages info, could you knit the new .Rmd and pull the Rmd and the pdf's to the master branch?
Let me know if you think is fine.

**Comment by anisfeld** [comment]
Closing addresses #12

**Pull request merged**